subprojects/platform-jvm/src/integTest/groovy/org/gradle/jvm/toolchain/JavaToolchainDownloadIntegrationTest.groovy

```diff
@@ -97,7 +95,6 @@ class JavaToolchainDownloadIntegrationTest extends AbstractIntegrationSpec {
         """
 
         propertiesFile << """
-            org.gradle.java.installations.auto-detect=false
```

Comment:
So this would be a test where we would not disable auto-download by default, but leave detection disabled, right?

Reply:
Exactly